### ISSUE TITLE
Payment Cannot Go Through

### ISSUE THREAD

**Issue by wilsonljx** [freethlua/individualogist-whats-your-archetype#43]

**Comment by wilsonljx** [comment]
Hi Freeth,

Have just sent you an e-mail regarding your PayPal account being unable to accept payments. Please look into that and let me know when that's sorted out, or if you have an alternative method of payment.

(edit: facebook issue moved to #44)

**Comment by wilsonljx** [comment]
<!--
In any case, we attempted to send you $800USD. 

Here's the message that we wanted to attach with the payment, for your reference:

Hi Freeth,

This is Aaron here. We hope that $800USD is enough to justify the talent and effort that you have invested into our project. Please let us know if this is an amount that you're happy with.

Thing is, we enjoy working with you. The way that you've structured everything, and how you've gone above and beyond to improve our funnel is greatly appreciated. You are indeed a talented developer, and you have done so much for us. We would like to work with you in the long term. Even though this project has already been completed, we'll be working constantly to improve it and make it better, both in terms of conversions and user experience.

That being said, it would be great if we could work closer together. I'm not sure if you're comfortable with this, but I believe our communication methods can be improved. If you disappeared somehow or some way, we would have no way of getting in touch with you. Perhaps you're just a private person by nature, but it would put us a great ease if you could furnish us with a contact number of some sort. And to ensure that we're not disrupting your day constantly, we could maybe set some rules of when we're able to get in touch with you.

Long story short, we like you, we want to work more with you, but we want to have a better way of communicating with you. 

Do let us know if that's possible, and also whether the $800 is enough. Thanks, Freeth!

**Comment by freethlua** [comment]
facebook comments issue fixed in #44. Please test it out. 

Will respond to above comment shortly in an email
